src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingVisitor.cs

```diff
@@ -132,12 +134,24 @@ public override void VisitMarkupBlock(MarkupBlockSyntax node)
         public override void VisitMarkupElement(MarkupElementSyntax node)
         {
             Visit(node.StartTag);
-            _currentIndentationLevel++;
+
+            // Temporary fix to not break the default Html formatting behavior
```

Comment:
I find it helpful to say what the temporary thing is locked behind. `Temporary fix to not break the default Html formatting behavior. Can be removed after (issue#|condition X is met).`